imports/plugins/core/orders/client/containers/orderSummaryContainer.js

```diff
@@ -116,25 +116,19 @@ class OrderSummaryContainer extends Component {
 }
 
 const composer = (props, onData) => {
-  const userId = Meteor.userId();
-  const userSub = Meteor.subscribe("UserProfile", userId);
   const orderSub = Meteor.subscribe("Orders");
-  let profile = {};
-
-  if (userSub.ready() && orderSub.ready()) {
-    if (typeof userId === "string") {
-      const userProfile = Accounts.findOne(userId);
-      if (!userProfile) {
-        return profile;
-      }
-      profile = userProfile.profile.addressBook[0];
-    }
 
+  if (orderSub.ready()) {
+    // Find current order
     const order = Orders.findOne({
       "_id": props.orderId,
       "shipping._id": props.fulfillment._id
     });
 
+    let profile = {};
```

Comment:
Yeah.... good question, definitely don't need to do that... 

was rushing through, just copy/pasting as much as needed, in the old version we could re-assign it a couple times and it was necessary. Updated to make more sense.